### PR TITLE
Import form.io forms directly to Formsflow, Metrics data overlaping i…

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**
…n pichart fix

# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-1759 
https://aottech.atlassian.net/browse/FWF-1884
Issue Type: BUG/ FEATURE

# Changes

Now the form.io forms can be directly imported to formsflow,
Metrics data overlaping issue fix.
image
 

# How has the change been tested?

# Screenshots (if applicable)

# Build Success screenshot (Till a CICD pipeline is set up)

# Notes

### CODE REVIEW

File: forms-flow-web/src/components/Form/List.js [hunk 1]
```diff
@@ -428,12 +428,26 @@ const List = React.memo((props) => {
...
 
+
   const fileUploaded = async (evt) => {
     FileService.uploadFile(evt, async (fileContent) => {
-      if (fileContent) {
-        dispatch(setFormUploadList(fileContent?.forms || []));
+      if ("forms" in fileContent) {
+        var formToUpload = fileContent;
+      }
+      else {
```
Comment (abhilash-aot) [inline forms-flow-web/src/components/Form/List.js:437]: Can reduce this to a single statement say
 let obj = {a: 1, b: 2, c: 3, d: 4, e: 5 };

['c', 'e'].forEach(e => delete obj[e]);

// obj is now {a:1, b:2, d:4}

Comment (abhilash-aot) [reply forms-flow-web/src/components/Form/List.js:437]: And try avoiding var . Use let or const more in blocks

**Review by abhilash-aot — CHANGES_REQUESTED** [review]

**Comment by sreehari-aot** [comment]
You can use the interactive terminal for commit messages, keep the first line short leave one line blank between the title and descriptions. Better if PR is for a single issue or feature at a time.

File: forms-flow-web/src/components/Form/List.js (hunk 1 above)
Comment (Josephalexantony-aot) [reply forms-flow-web/src/components/Form/List.js:437]: sure

**Pull request closed without merging**